### ISSUE TITLE
Error 403 in AWS EB multi container docker environment

### ISSUE THREAD

**Issue by mohideen** [tawan/active-elastic-job#94]
When running in docker, sqs_message_consumer expects the cgroup to contain the word `docker` but that is not present in the current solution stack.
https://github.com/tawan/active-elastic-job/blob/0fc32d5969879dbdde1c94473ba67fd5e77a5192/lib/active_elastic_job/rack/sqs_message_consumer.rb#L125

Output of `cat /proc/1/cgroup`
```
9:perf_event:/ecs/b15c7c5b-3b06-446d-a56d-54b3ffd91c08/dbcd08cd614c970a01cc3f3d07fa9517153f621b4bfb30439734493098b3a46a
8:memory:/ecs/b15c7c5b-3b06-446d-a56d-54b3ffd91c08/dbcd08cd614c970a01cc3f3d07fa9517153f621b4bfb30439734493098b3a46a
7:hugetlb:/ecs/b15c7c5b-3b06-446d-a56d-54b3ffd91c08/dbcd08cd614c970a01cc3f3d07fa9517153f621b4bfb30439734493098b3a46a
6:freezer:/ecs/b15c7c5b-3b06-446d-a56d-54b3ffd91c08/dbcd08cd614c970a01cc3f3d07fa9517153f621b4bfb30439734493098b3a46a
5:devices:/ecs/b15c7c5b-3b06-446d-a56d-54b3ffd91c08/dbcd08cd614c970a01cc3f3d07fa9517153f621b4bfb30439734493098b3a46a
4:cpuset:/ecs/b15c7c5b-3b06-446d-a56d-54b3ffd91c08/dbcd08cd614c970a01cc3f3d07fa9517153f621b4bfb30439734493098b3a46a
3:cpuacct:/ecs/b15c7c5b-3b06-446d-a56d-54b3ffd91c08/dbcd08cd614c970a01cc3f3d07fa9517153f621b4bfb30439734493098b3a46a
2:cpu:/ecs/b15c7c5b-3b06-446d-a56d-54b3ffd91c08/dbcd08cd614c970a01cc3f3d07fa9517153f621b4bfb30439734493098b3a46a
1:blkio:/ecs/b15c7c5b-3b06-446d-a56d-54b3ffd91c08/dbcd08cd614c970a01cc3f3d07fa9517153f621b4bfb30439734493098b3a46a
```

Solution Stack: `64bit Amazon Linux 2018.03 v2.11.2 running Multi-container Docker 18.03.1-ce (Generic)`

**Comment by joeyparis** [comment]
I'm having a similar issue with the AWS EB single container docker environment. Did you ever find a work around for this?

**Comment by mohideen** [comment]
@joeyparis I created a PR to fix it #107. As it was not merged, I ended up using my own fork.
You can test using my fork using:
```
gem 'active_elastic_job', git: 'https://github.com/mohideen/active-elastic-job.git', branch: 'feature/LIBBCM-131'
```
You may want to create your own fork.
